step-templates/proxmox-deploy-lxc.json

```diff
@@ -1,194 +1,194 @@
-		},
-		{
-			"Id": "0a13d075-3a2f-4718-9793-f9b2f5963455",
-			"Name": "Proxmox.LXC.OSTemplate",
-			"Label": "LXC OS Template",
-			"HelpText": "The template image or backup image for the LXC",
-			"DefaultValue": "",
-			"DisplaySettings": {
-				"Octopus.ControlType": "SingleLineText"
-			}
-		},
-		{
-			"Id": "55f7937b-4a5f-49bb-b129-f589c3ce51cc",
-			"Name": "Proxmox.LXC.Password",
-			"Label": "LXC Root Password",
-			"HelpText": "This will be the root password once the LXC container has been created",
-			"DefaultValue": "",
-			"DisplaySettings": {
-				"Octopus.ControlType": "Sensitive"
-			}
-		},
-		{
-			"Id": "671751a5-d213-445a-a73f-dd9f1f33d754",
-			"Name": "Proxmox.LXC.Storage",
-			"Label": "LXC Storage",
-			"HelpText": "Where the rootfs for this LXC will be stored.",
-			"DefaultValue": "local",
-			"DisplaySettings": {
-				"Octopus.ControlType": "SingleLineText"
-			}
-		},
-		{
-			"Id": "e796b021-0c31-4e03-8825-23c19cbf8876",
-			"Name": "Proxmox.LXC.Network",
-			"Label": "LXC Networks",
-			"HelpText": "The list of network connections this LXC has. Each network connection on a new line.\n\n\n`name=<name>,bridge=<bridge>,firewall=<0|1>,gw=<IPv4>,ip=<IPv4/CIDR|dhcp|manual>`\n\n\nMore Info: [https://pve.proxmox.com/pve-docs/api-viewer/#/nodes/{node}/lxc](https://pve.proxmox.com/pve-docs/api-viewer/#/nodes/{node}/lxc)",
-			"DefaultValue": "name=eth0,bridge=vmbr0",
-			"DisplaySettings": {
-				"Octopus.ControlType": "MultiLineText"
-			}
-		},
-		{
-			"Id": "c9e865ea-a7b2-42d3-aca6-8772e54b893a",
-			"Name": "Proxmox.LXC.Cores",
-			"Label": "LXC CPU Cores",
-			"HelpText": "The amount of CPU cores the LXC is assigned",
-			"DefaultValue": "1",
-			"DisplaySettings": {
-				"Octopus.ControlType": "SingleLineText"
-			}
-		},
-		{
-			"Id": "74b64687-8aed-433d-bce3-714bfd738927",
-			"Name": "Proxmox.LXC.Memory",
-			"Label": "LXC Memory",
-			"HelpText": "The amount of Memory the LXC is assigned.",
-			"DefaultValue": "2048",
-			"DisplaySettings": {
-				"Octopus.ControlType": "SingleLineText"
-			}
-		},
-		{
-			"Id": "85c52eb6-a25f-4f8e-a068-2a165160d94d",
-			"Name": "Proxmox.LXC.RootSize",
-			"Label": "LXC Rootfs Size",
-			"HelpText": "The size of the root volume for this LXC, Size is in GB",
-			"DefaultValue": "60",
-			"DisplaySettings": {
-				"Octopus.ControlType": "SingleLineText"
-			}
-		},
-		{
-			"Id": "fd072ced-1c8b-4205-a724-e9456b5152c6",
-			"Name": "Proxmox.LXC.StartOnCreate",
-			"Label": "LXC Start Once Created",
-			"HelpText": "Should the LXC start once the LXC has been created",
-			"DefaultValue": "False",
-			"DisplaySettings": {
-				"Octopus.ControlType": "Checkbox"
-			}
-		},
-		{
-			"Id": "dfea6eea-47ff-4489-9434-8ff80bbb8694",
-			"Name": "Proxmox.LXC.Force",
-			"Label": "LXC Overwrite Container",
-			"HelpText": "Overwrites an existing LXC with the same VMID",
-			"DefaultValue": "False",
-			"DisplaySettings": {
-				"Octopus.ControlType": "Checkbox"
-			}
+	  {
+		"Id": "",

```

Comment:
All of the parameter Ids have been removed. Please reinstate. If you create a new one, that's also ok, but it should have a unique guid value for its id.